cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: cluster-service-admin
+objects:
+  - apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: cluster-service-admin
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: cluster-service-mgmt
+      namespace: cluster-service-admin
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: namespace-admin
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - namespace
```

Comment:
Does the cluster role need to include: serviceaccounts and tokens?

Reply:
I guess `cluster-admin` would be too much for our case, or given that it's a test environment, is it reasonable?

Reply:
Tried it locally using minikube with 1.31, there it was not included in admin

Reply:
I think clusteradmin would be to wide

Reply:
Thanks for the clarification:)